### ISSUE TITLE
Possible normalization error in Gram-Schmidt orthonormalization process in GPNR

### ISSUE THREAD

**Issue by MaxtirError** [google-research/google-research#1881]
I am using your open-source code and I found a possible issue that I would like to share with you and hope to get your answer. The issue occurs in the gpnr.py file, lines 300-309, as follows:
~~~python
    # Get the up vector
    upv = camtoworld[Ellipsis, :3, 1]
    # Get the ray direction.
    raydir = batch_rays.directions
    rdotup = (raydir * upv).sum(-1, keepdims=True)
    orthoup = upv - rdotup * raydir
    orthoup = orthoup / (orthoup**2).sum(-1, keepdims=True)
    vec0 = jnp.cross(orthoup, raydir)
    vec0 = vec0 / (vec0**2).sum(-1, keepdims=True)
    # Stack it such that the matrix is the transpose (inverse).
    r_relative = jnp.stack([vec0, orthoup, raydir], axis=1)
~~~

What I think the issue is: This section theoretically does a Gram-Schmidt orthonormalization process, but I think it should use norm to normalize the vectors, instead of the squared sum as written in the code, that is, I suggest changing `(orthoup**2).sum(-1, keepdims=True) ~and~ (vec0**2).sum(-1, keepdims=True)` to `torch.norm(orthhoup, dim=-1, keepdims=True)`.

Thank you very much for your excellent work, I look forward to your reply, thank you for your consideration.
